src/TTextEdit.cpp

```diff
@@ -794,14 +799,16 @@ void TTextEdit::unHighlight(QRegion& region)
         if (y == y1) {
             x = mPA.x();
         }
+
+        if (y >= static_cast<int>(mpBuffer->buffer.size())) {
+            break;
+        }
+        mpBuffer->dirty[y] = true;
```

Comment:
...Yeah, this line is premature after all for the same reason as the `highlight` case ...